library/gradle.properties

```diff
@@ -1,5 +1,2 @@
 VERSION_NAME=0.9.0-SNAPSHOT
-POM_ARTIFACT_ID=mapbox-android-gestures
-POM_NAME=Mapbox Android Gestures Library
-POM_DESCRIPTION=Mapbox Android Gestures Library
-POM_PACKAGING=aar
\ No newline at end of file
+POM_NAME=Mapbox Android Gestures Library
```

Comment:
I'm not that familiar with it but all our Maps SDK `gradle.properties` files have more properties:
```
POM_GROUP_ID=
POM_ARTIFACT_ID=
POM_ARTIFACT_TITLE=
POM_DESCRIPTION=
REGISTRY_SDK_NAME=
```

Reply:
Yeah, that's because this properties are then reused in the [gradle config](https://github.com/mapbox/mapbox-maps-android/blob/main/gradle/artifact-settings.gradle#L2). 
In this repo config is not reused so it's inlined in https://github.com/mapbox/mapbox-gestures-android/pull/103/files#diff-226f5cade15955b90742f0e9d012e5e355fca535a25ba3764dfba1fa1846836cL2.